projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -563,6 +563,7 @@ function render_block( $attributes ) {
 		),
 		'subscribe_placeholder'  => get_attribute( $attributes, 'subscribePlaceholder', esc_html__( 'Type your email…', 'jetpack' ) ),
 		'submit_button_text'     => get_attribute( $attributes, 'submitButtonText', $is_paid_subscriber ? esc_html__( 'Upgrade', 'jetpack' ) : esc_html__( 'Subscribe', 'jetpack' ) ),
+		'subscribed_button_text' => get_attribute( $attributes, 'subscribedButtonText', esc_html__( '✓ Subscribed', 'jetpack' ) ),
```

Comment:
Thinking if this should be `'✓ ' . esc_html__( 'Subscribed', 'jetpack' )` ?

Reply:
if we are going to allow editing this in the future it could be good to add the flexibility of allowing to remove the ✓

Reply:
I agree — we should include the ✓ in string they can edit, but for translation purposes I'm not sure if it makes a difference to send it for translations?

Reply:
Just `$subscribed = '✓ ' . esc_html__( 'Subscribed', 'jetpack' );` should work and the checkmark would be displayed correctly for RTL as long as `direction: rtl` is set.

Reply:
ok! Done https://github.com/Automattic/jetpack/pull/33897/commits/1be309ea9fb3e3408c315c78b6ed3a1b2e803934